benchmark/benchmark_turbomind_engine_a100.sh

```diff
@@ -0,0 +1,32 @@
+# #!/bin/bash
+
+dataset_path="benchmark/ShareGPT_V3_unfiltered_cleaned_split.json"
+########################################## TurboMind engine: fp16 or bf16 ##########################################
+## 7B. gemm_tune -> profile_throughput
+tp=1
+cache_max_entry_count=0.95
```

Comment:
Should we support this in torch engine.

Reply:
I think we should. May pull another PR to support it.